Add model_id to device filters (#36926)
home-assistant/home-assistant.github.io: source/_docs/blueprint/selectors.markdown

        the set model.
      type: string
      required: false
    model_id:
      description: >

>         When set, it limits the list of areas that provide devices that have
>         the set model ID.

Reviewer: ```suggestion
        When set, the list of areas is limited to areas with devices that have
        the set model ID.
```

        When set, it limits the list of devices to devices that have the set model.
      type: string
      required: false
    model_id:
      description: >

>         When set, it limits the list of devices to devices that have the set model ID.

Reviewer: ```suggestion
        When set, the list of devices is limited to devices that have the set model ID.
```

        the set model.
      type: string
      required: false
    model_id:
      description: >

>         When set, the list only includes floors that have devices which have
>         the set model ID.

Reviewer: ```suggestion
        When set, the list only includes floors with devices that have
        the set model ID.
```

      description: When set, it limits the targets to devices by the set model.
      type: string
      required: false
    model_id:

>       description: When set, it limits the targets to devices by the set model ID.

Reviewer: ```suggestion
      Description: When set, the targets are limited to devices that have the set model ID.
```

>       Description: When set, the targets are limited to devices that have the set model ID.

Reviewer: ```suggestion
      description: When set, the targets are limited to devices that have the set model ID.
```
